### PR TITLE
new 4 mongodb init havron

### PR DESCRIPTION
**Pull request by OleksandrHavron**
MongoDB initialization

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,8 +2,10 @@ asgiref==3.4.1
...
 django-filter==2.4.0
 djangorestframework==3.12.4
+djongo==1.3.6
 Markdown==3.3.4
 psycopg2==2.9.1
+pymongo==3.12.0
 python-decouple==3.4
 pytz==2021.1
-sqlparse==0.4.1
+sqlparse==0.2.4
```
Comment (MrSancho42) [inline requirements.txt:11]: Katya said that we need to install the latest version of Python, Django and React. So I think we also need to use the latest versions of packages.
Or are there any problems with the new version?

Comment (OleksandrHavron) [reply requirements.txt:11]: I received this message when i tried to install newer version of sqlparse: 'djongo requires sqlparse==0.2.4, but you have sqlparse 0.4.1 which is incompatible.' Djongo is connector which let's you use Django with MongoDB

Comment (MrSancho42) [reply requirements.txt:11]: Ok, then we need to use sqlparse 0.2.4

**Review by MrSancho42 — APPROVED** [review]

File: codeArena/codeArena/settings.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+# Database
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+MONGODB_USER = config("MONGODB_USER")
+MONGODB_USER_PASS = config("MONGODB_USER_PASS")
+MONGODB_HOST = config("MONGODB_HOST")
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'djongo',
```
Comment (Viktorika-p) [inline codeArena/codeArena/settings.py:84]: You should rename 'djongo' on 'django'.

Comment (OleksandrHavron) [reply codeArena/codeArena/settings.py:84]: Djongo is connector which let's you use Django with MongoDB

**Review by Viktorika-p — APPROVED** [review]

**Review by dmytrosmetanin — APPROVED** [review]
approved

**Pull request merged**